clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -709,13 +721,17 @@ def write_bids(
             os.makedirs(to / (Path(bids_full_path).parent))
         except OSError:
             pass
-        run_dcm2niix(
+        dcm2niix_success = run_dcm2niix(
             Path(metadata["source_path"]).parent,
             to / str(Path(bids_full_path).parent),
             metadata["bids_filename"],
             True,
         )
-        if "dwi" in metadata["bids_filename"] and metadata.manufacturer == "Philips":
+        if (
+            "dwi" in metadata["bids_filename"]
+            and "Philips" in metadata.manufacturer
```

Comment:
Why the change from equality to a weaker inclusion here ?

Reply:
Sometimes the manufacturer is not just `Philips`but `Philips &Co`or whatnot